cmd/wksctl/kubeconfig/kubeconfig.go

```diff
@@ -55,7 +56,7 @@ func init() {
 		"Branch within git repo containing your cluster and machine information")
 	Cmd.Flags().StringVar(&kubeconfigOptions.gitPath, "git-path", ".", "Relative path to files in Git")
 	Cmd.Flags().StringVar(&kubeconfigOptions.gitDeployKeyPath, "git-deploy-key", "", "Path to the Git deploy key")
-	Cmd.Flags().StringVar(&kubeconfigOptions.sshKeyPath, "ssh-key", "./cluster-key", "Path to a key authorized to log in to machines by SSH")
+	Cmd.Flags().StringVar(&kubeconfigOptions.sshKeyPath, "ssh-key", "./setup/cluster-key", "Path to a key authorized to log in to machines by SSH")
```

Comment:
I don't think this is _quite_ the right fix since the "setup" directory is specific to WKP. WKSCtl doesn't know anything about a setup dir. If we were going to make _this_ change, it should probably be in the WKS repo. However, we might be able to finesse this by changing the behavior to search the directory tree below the user's current location for the files (if path left blank -- the default would no longer be "./cluster-key"). That way the existing code would continue to work everywhere while still fixing the WKP issue. What do you think? BTW, one of the integration tests failed due to this change; take a look at the results of "integration-tests-container".